src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1296,18 +1329,28 @@ InferenceProfiler::SummarizeLatency(
 
   std::tie(summary.client_stats.avg_latency_ns, summary.client_stats.std_us) =
       GetMeanAndStdDev(latencies);
+  std::tie(
+      summary.client_stats.avg_response_latency_ns,
+      summary.client_stats.std_us) = GetMeanAndStdDev(response_latencies);
 
   // retrieve other interesting percentile
   summary.client_stats.percentile_latency_ns.clear();
+  summary.client_stats.response_percentile_latency_ns.clear();
   std::set<size_t> percentiles{50, 90, 95, 99};
+  std::set<size_t> response_percentiles{50, 90, 95, 99};
   if (extra_percentile_) {
     percentiles.emplace(percentile_);
+    response_percentiles.emplace(percentile_);
   }
 
   for (const auto percentile : percentiles) {
     size_t index = (percentile / 100.0) * (latencies.size() - 1) + 0.5;
```

Comment:
Should you calculate a `response_index` in case `latencies.size() != response_latencies.size()`? Would they ever be unequal?

Reply:
Nope this is a good catch. I need to calculate the response index.

Reply:
Updated with an index calculation for the response_latencies.